plugins/repository-azure/build.gradle

```diff
@@ -48,6 +48,13 @@ versions << [
 
 dependencies {
   api project(path: 'azure-storage-blob', configuration: 'shadow')
+  if (isEclipse) {
```

Comment:
Ug, but I don't see a better option here. Another reason we should look at reimplementing these "hacks" as artifact transforms.

Reply:
IntelliJ has something nicer but I don't know that eclipse could integrate with this. I really wish we had an eclipse user among the delivery folks because they'd have the time to dedicate to understanding. Its difficult being the odd one out.